tabs/tab_survival.py

```
@@ -268,244 +241,484 @@ def surv_ui(namespace: str) -> ui.TagChild:
-    
-    # ==================== REACTIVE STATES ====================
-    
-    surv_df_current = reactive.Value(None)  # Current dataset (original or matched)
-    curves_result = reactive.Value(None)    # KM/NA analysis result
-    landmark_result = reactive.Value(None)  # Landmark analysis result
-    cox_result = reactive.Value(None)       # Cox regression result
+# ==============================================================================
+# Server Logic - Modern Module Pattern
+# ==============================================================================
+@module.server
+def survival_server(input, output, session, df: reactive.Value, var_meta: reactive.Value,
```

Ruff (0.14.10)

Unused function argument: `output`

(ARG001)